runtimes/pythonrt/py-sdk/autokitteh/events.py

```diff
@@ -9,7 +9,7 @@
 def subscribe(connection_name: str, filter: str) -> str:
     """Subscribe to events on connection. Optional filter is a CEL expression."""
     # Dummy implementation for local development.
-    return uuid4().hex
+    return f"sig_{uuid4().hex}"
```

Comment:
should be typeid

Reply:
These are dummy functions that are replaced by the runner to call `ak`. I don't think adding `typeid-python` as dependency to SDK just to be compatible is worth it. Same for below.